portal/src/routes/(authenticated)/projects/[filter=projectSelector]/[id]/+page.server.ts

```diff
@@ -113,15 +113,19 @@ export const load = (async ({ params, locals }) => {
 
 export const actions: Actions = {
   page: async function ({ params, request, locals }) {
-    const userId = (await locals.auth())?.user.userId;
-    if (!userId) return error(400);
+    const user = (await locals.auth())?.user;
+    if (!user) return error(401);
+
+    const orgId = parseInt(params.id!);
 
     const form = await superValidate(request, valibot(projectSearchSchema));
     if (!form.valid) return fail(400, { form, ok: false });
+    // if user modified hidden values
+    if (orgId !== form.data.organizationId) return fail(403, { form, ok: false });
```

Comment:
Here we should just use the `orgId` variable and leave `orgId` out of the paging schema.